pkg/proc/mem.go

```diff
@@ -35,7 +35,12 @@ type memCache struct {
 }
 
 func (m *memCache) contains(addr uint64, size int) bool {
-	return addr >= m.cacheAddr && addr <= (m.cacheAddr+uint64(len(m.cache)-size))
+	end := addr + uint64(size)
+	if end < addr {
```

Comment:
This branch is never exercised by any test. Can you add a case for it?

Reply:
Done, please take a look again.